.gitignore

```diff
@@ -53,6 +54,7 @@ magic.odm.yaml
 .project
 .pydevproject
 .settings
+/install/linux/usr/share/odemis/sim/meteor-reverse-sim.odm.yaml
```

Comment:
Let's drop the changes on .gitignore. The `htmlcov/` should already be there.

Reply:
The PEP 8 standard for docstrings refers to PEP 257 and in there it describes multiline docstrings with an example:
def bar():
    """
    A multi-line
    docstring.
    """
So I am a little confused. In my mind I wanted to start that line just below the first part with the same indentation.
If you say we always have to use 2 spaces in this case, we might also add that to Karishma's PR for coding guidelines.